src/main/resources/operator/aws-eks/aws-route53-record-update.json

```diff
@@ -0,0 +1,14 @@
+{
+  "Comment": "Updating Alias resource record sets in Route 53",
```

Comment:
It's too generic.
It's better to be more precise.
What is it? Deploy, clustered, etc. Please make it more specific, what for we are doing it.

Reply:
done